### ISSUE TITLE
The newline_behavior attribute is not working as expected in inline editors.

### ISSUE THREAD

**Issue by muhammadmohie** [tinymce/tinymce#8040]

**Comment by muhammadmohie** [comment]
You are right but I tried to replace `newline_behavior: "default"` with `newline_behavior: "invert"` to swap Enter and Shift+Enter behaviors, but with no effect applied.

**Comment by muhammadmohie** [comment]
Any updates? @tiny-james?

**Comment by TheSpyder** [comment]
That does seem like a bug, given that shift+enter works with `default` but enter doesn't work with `invert`. Thanks for the report, we will investigate.

**Comment by muhammadmohie** [comment]
Thanks a lot for your time, waiting for the fix.

**Comment by micaelpreis** [comment]
I'm also the same issue, I'm using a span and the newline_behaviour is also not working as expected there.

**Comment by TheSpyder** [comment]
We've tentatively scheduled the fix for 6.3, due next month, but we haven't confirmed whether it will make it into that release yet.

**Comment by TheSpyder** [comment]
It didn't fit into the 6.3 release and it hasn't been rescheduled yet. Sorry for the lack of updates, I labelled our internal report incorrectly and it wasn't properly connected to this issue.

**Comment by DGovender** [comment]
Hi @TheSpyder, has this fix been scheduled for any release as this issue is still open?

**Comment by DAdq26** [comment]
Hi 

for us when using newline_behavior: 'linebreak' to avoid new paragraph after hiting enter, this option breaks key Enter with list
(with version 7.1.1)

Nor enter nor shift enter allows you to add a new list item
It's a real issue for us

![image](https://github.com/tinymce/tinymce/assets/134490838/9000e38c-b214-4d29-8da4-53d6d0750184)

Best

**Comment by dikowski** [comment]
I'm in the same situation as @DAdq26. newline_behavior: 'linebreak' messes up the ordered and unordered list functionality. This is a problem. Hoping a solution is coming soon.

**Comment by TheSpyder** [comment]
This is getting off topic. The issue is regarding inline editors; please log a new issue if you need to discuss other things.

`newline_behavior: 'linebreak'` [completely disables the action normally assigned to the enter key](https://www.tiny.cloud/docs/tinymce/latest/content-behavior-options/#newline_behavior). If you want shift+enter to insert a new block tag, use `newline_behavior: 'invert'`. There is no "solution" for lists when you use `linebreak`. The same thing will happen in all block tags including headings. It's what you ask for when you configure the `newline_behavior` setting that way.

**Comment by DAdq26** [comment]
Hi,

No problem, i will open a new issue.
But sure the behaviour is wrong with newline is setted on linebreak, if you try to use simply ordered list you are block hitting enter, no new item list is created and you can not leave the list when hitting twice

Best

**Comment by DAdq26** [comment]
@see 9771
